DataPublishingNotAvailableError
===============================

.. currentmodule:: pywwt

.. autoexception:: DataPublishingNotAvailableError
